schema.sql

```diff
@@ -10,4 +10,25 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(100);
\ No newline at end of file
+ADD COLUMN species VARCHAR(100);
+
+CREATE TABLE owners (
+    ID INT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    full_name VARCHAR(100) NOT NULL,
+    age INT
+);
+
+CREATE TABLE species (
+    ID INT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR(100) NOT NULL
+);
+
+ALTER TABLE animals 
+    ADD COLUMN species_id INT,
+    ADD CONSTRAINT fk_species
+        FOREIGN KEY (species_id) REFERENCES species(id);
```

Comment:
- [ ] Ouch! you seem to be missing removing the `species` column that was added at line `13` before adding the `species_id` column here so please make sure to use `DROP` statement to remove that column.